index.js

```diff
@@ -84,18 +84,88 @@ module.exports = {
     }
   },
 
+  _shouldIncludeHelpers() {
+    let appOptions = (this.app && this.app.options) || {};
+    let customOptions = appOptions['ember-cli-babel'];
+
+    if (customOptions && 'includeExternalHelpers' in customOptions) {
+      return customOptions.includeExternalHelpers === true;
+    } else {
+      return false;
+    }
+  },
+
+  _importHelpers(app) {
+    let helpersPath = 'vendor/babel-helpers/babel-helpers.js';
+
+    if (this.import) {  // support for ember-cli >= 2.7
```

Comment:
ember-cli-babel@7 explicitly only supports ember-cli@2.13 and higher (also documented in the README), these conditionals can be removed...